source/_integrations/roborock.markdown

```diff
@@ -238,3 +238,9 @@ Roborock servers require accepting a user agreement before using the API, which
 3. Hit **Revoke authorization**.
 4. Log back in and accept the policy.
 5. Reload the Roborock integration!
+
+### The integration tells me it cannot reach my vacuum and is using the cloud API and that this is not supported
+
+This integration has the capability to control your devices through the cloud API and the local API. If the local api is not reachable, it will just use the cloud API. We recommend only using the local API as it helps prevent any kind of rate-limiting.
```

Comment:
```suggestion
This integration has the capability to control your devices through the cloud API and the local API. If the local API is not reachable, it will just use the cloud API. We recommend only using the local API as it helps prevent any kind of rate-limiting.
```